uikit/src/main/java/com/midtrans/sdk/uikit/PaymentMethods.java

```diff
@@ -155,6 +155,10 @@ public static BankTransfer createBankTransferModel(Context context, String type,
                     bankTransfer = new BankTransfer(type, context.getString(R.string.bni_bank_transfer), R.drawable.ic_bni, 4, context.getString(R.string.payment_bank_description_bni), status);
                     break;
 
+                case PaymentType.BRI_VA:
+                    bankTransfer = new BankTransfer(type, context.getString(R.string.bri_bank_transfer), R.drawable.ic_bri, 4, context.getString(R.string.payment_bank_description_bri), status);
```

Comment:
Is this have same priority with BNI? (4)

Reply:
updated the priority please re-check @farhanpahlevi